### PR TITLE
add e2e test for default icons 

### LINKED ISSUE #13451: [QA][e2e] Create new e2e test to cover change in a feature for selecting identicon type
Acceptance criteria:
* Develop a new e2e test to cover new identicon switch controls
* Put test code into the same place as it was for the previous control

Related ticket: [https://github.com/MetaMask/metamask-extension/issues/13264](https://github.com/MetaMask/metamask-extension/issues/13264)

Target screen:
148792311-786cad53-d1eb-4b32-932b-5d4d4b017d90

### PR DESCRIPTION
**Pull request by mirjanaKukic**
Resolves [#13451](https://github.com/MetaMask/metamask-extension/issues/13451)
This covers features merged in https://github.com/MetaMask/metamask-extension/pull/13408

### CODE REVIEW

**Review by seaona — COMMENTED** [review]
That's a great testcase  @mirjanaKukic ! 

I have one proposal for a simple change: move testing Account Identicon into a file called `settings-general.spec.js`. This way, testcase is more organized and we can expand it with other testcases contained inside **Settings General** view, like Currency Conversion, Primary Currency etc.

Another main reason for that is because **3box** is being removed and replaced soon in these 2  PRs:
- https://github.com/MetaMask/metamask-extension/pull/14571
- https://github.com/MetaMask/metamask-extension/pull/15243

which means that we would need to remove the `threebox.spec.js` file and modify the testcases to adapt to its replacement function (Backup and Restore data). 
@PeterYinusa I've created a [ticket](https://github.com/MetaMask/metamask-extension/issues/15412) for 3box removal and update on Zenhub QA Backlog

File: test/e2e/fixtures/threebox-enabled/state.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
       "tokens": [],
       "hiddenTokens": [],
       "suggestedTokens": {},
-      "useBlockie": true,
+      "useBlockie": false,
```
Comment (PeterYinusa) [inline test/e2e/fixtures/threebox-enabled/state.json:100]: I don't believe we need this change to the threebox fixture

**Review by PeterYinusa — CHANGES_REQUESTED** [review]

File: test/e2e/fixtures/threebox-enabled/state.json (hunk 1 above)
Comment (mirjanaKukic) [reply test/e2e/fixtures/threebox-enabled/state.json]: Yes, you're correct. I fixed it.

**Review by darkwing — APPROVED** [review]

**Review by digiwand — APPROVED** [review]
Looks like the changes @seaona and @PeterYinusa requested were made. LGTM

**Review by PeterYinusa — APPROVED** [review]

**Pull request merged**